src/mono/wasm/runtime/workers/dotnet-crypto-worker.js

```diff
@@ -379,16 +380,20 @@ function _stringify_err(err) {
     return (err instanceof Error && err.stack !== undefined) ? err.stack : err;
 }
 
-var s_channel;
-
-setup_proxy_console("crypto-worker", console, self.location.origin);
+let s_channel;
 
 // Initialize WebWorker
 onmessage = function (p) {
-    var data = p;
+    /**
+     * @type {any}
+     */
+    let data = p;
     if (p.data !== undefined) {
         data = p.data;
     }
+    if (Configuration === "Debug") {
```

Comment:
This should depend on a setting which flows in from the config, so it can be enabled via the host, xharness etc.

Reply:
which setting do you propose ? I did this in order to trim the `Release` size.

Reply:
Because this is copy of the code, because it's in separate target js file.

Reply:
Also, do we really want to have log forwarding in the production code ? I think it has security implications.

Reply:
I did `if (data.config.debug_level)` for now and fixed the startup to call `init_crypto` after the config was loaded already.